portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,45 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  /* margin-left: auto;
+  margin-right: auto; */
+  width: 100%;
 }
 
-#greeting-container {
+/* #greeting-container {
   margin-top: 20px;
+} */
+
+body {
+  background-image: url('images/background4.jpeg');
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+}
+
+h1{
+    padding: 50px;
+    text-align: center;
+    top:0;
+    right:0;
+    /* background-color: white; */
+    color: white;
+    font-size: 70px;}
```

Comment:
Move } to the next line. Also order the properties alphabetically. Add space after : so top: 0 instead of top:0

Reply:
done